packages/prisma/seed-utils.ts

```diff
@@ -490,6 +500,21 @@ export async function seedRoutingForms(
           queryValue: {
             id: "aaba9988-cdef-4012-b456-719300f53ef8",
             type: "group",
+            children1: {
+              "b98b98a8-0123-4456-b89a-b19300f55277": {
+                type: "rule",
+                properties: {
+                  field: seededForm.formFieldSkills.id,
+                  value: [
+                    formFieldSkillsOptions.filter((opt) => opt.label === "JavaScript").map((opt) => opt.id),
+                  ],
+                  operator: "multiselect_equals",
+                  valueSrc: ["value"],
+                  valueType: ["multiselect"],
```

Comment:
Route was wrong. I had no form field conditions and thus Route-1 was always matching. It fixes it.